core/src/main/scala/com/nvidia/spark/rapids/tool/tuning/AutoTuner.scala

```diff
@@ -657,11 +657,17 @@ class AutoTuner(
   def calculateJobLevelRecommendations(): Unit = {
     // TODO - do we do anything with 200 shuffle partitions or maybe if its close
     // set the Spark config  spark.shuffle.sort.bypassMergeThreshold
-    getShuffleManagerClassName match {
-      case Right(smClassName) => appendRecommendation("spark.shuffle.manager", smClassName)
-      case Left(comment) => appendComment("spark.shuffle.manager", comment)
+    val isDynamicAllocationEnabled = appInfoProvider.getSparkProperty(
+      "spark.dynamicAllocation.enabled").contains("true")
+    if (isDynamicAllocationEnabled) {
+      comments += ProfilingAutoTunerConfigsProvider.shuffleManagerCommentForDynamicAllocation
```

Comment:
is this supposed to be `appendComment` instead of `comments +=` ? We try to standardize using `appendComment` because we can later piggiy-back on that method to pre-process evetything that's being added with less code changes.